torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -7074,9 +7074,9 @@ def test_ddp_inference(self):
             syncbn_model = nn.SyncBatchNorm(
                 2, momentum=0.99, track_running_stats=False
             ).cuda()
-            local_syncbn_model = copy.deepcopy(model)
+            local_syncbn_model = copy.deepcopy(syncbn_model)
```

Comment:
made some additional change to distributed_tests which I think makes more sense given the logic. please kindly take a look @rohan-varma

Reply:
oops. seems like after the change it starts to timeout.

Reply:
Hm, can you run it locally or does it hang? Anyway, that's outside of scope for this PR, previously we weren't testing syncbn, only pretending to, so it's ok to leave it as a followup.

Reply:
yes it hangs locally as well. skipping and filing an issue